make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -693,6 +703,20 @@ ifeq ($(ENABLE_HEADLESS_ONLY), false)
     endif
   endif
 
+  # The external libpng submitted in the jdk is an old version
+  # which does not contain .png_init_filter_functions_vsx.
+  # Therefore we need to disable PNG_POWERPC_VSX_OPT explicitely by setting
```

Comment:
explicitely -> explicitly

And it isn't an old version. It is very current.
I think we just chose not to import functionality we weren't using.

Reply:
Was fixed.